### PR TITLE
Add assets package and icons

### PR DESCRIPTION
**Pull request by allanpope**
* Add assets package. This contains custom Qantas assets such as the pebbles and roo.
* Add custom ribbon icon to a Qantas category.

### CODE REVIEW

File: packages/assets/src/assets/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import roo from './roo.svg';
+import pebble from './pebble.svg';
+import pebbleRed from './pebbleRed.svg';
+
+export {
+  roo,
+  pebble,
+  pebbleRed,
+};
```
Comment (ehyland) [inline packages/assets/src/assets/index.js]: You can do this in three lines:
```
export { default as roo } from './roo.svg';
export { default as pebble } from './pebble.svg';
export { default as pebbleRed } from './pebbleRed.svg';
```

**Comment by ehyland** [comment]
Great Idea

File: packages/assets/src/assets/index.js (hunk 1 above)
Comment (allanpope) [reply packages/assets/src/assets/index.js]: 👍 good point

**Comment by dougmcclurg** [comment]
Tight!

**Pull request merged**